LICENSE-3rdparty.md

```diff
@@ -275,3 +275,9 @@ Taskflow
   - https://github.com/taskflow/taskflow
 - Copyright: Dr. Tsung-Wei Huang
 - Usage: Threadpool implementation for batch processing.
+
+StainTools
+- License: MIT License
+  - https://github.com/Peter554/StainTools/blob/master/LICENSE.txt
```

Comment:
It would be nice if you could copy & paste the raw text to 3rdparty/LICENSE.StainTools.

Reply:
Done, thanks!